e2e/e2etests/test_precompiles_distribute.go

```diff
@@ -0,0 +1,196 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	"github.com/zeta-chain/node/precompiles/bank"
+	"github.com/zeta-chain/node/precompiles/staking"
+	ptypes "github.com/zeta-chain/node/precompiles/types"
+)
+
+func TestPrecompilesDistribute(r *runner.E2ERunner, args []string) {
+	require.Len(r, args, 0, "No arguments expected")
+
+	var (
+		spenderAddress            = r.EVMAddress()
+		distributeContractAddress = staking.ContractAddress
+		feeCollectorAddress       = authtypes.NewModuleAddress("fee_collector")
```

Comment:
We don't have a const somewhere for `fee_collector`?

Reply:
Modified in https://github.com/zeta-chain/node/commit/06d0c5d5a8a2d7ce54f84a666c7bef7784963ccf, where it was added to E2ERunner.